ts/packages/cli/src/commands/export.ts

```diff
@@ -156,6 +161,18 @@ export default class Export extends BaseCommand<typeof Export> {
     this.printStdout(formattedConfigs);
   }
 
+  keysMutations() {
+    if ([this.flags.prefix, this.flags.suffix].some((flag) => flag !== undefined)) {
```

Comment:
what if one of them is undefined?

<img width="128" alt="image" src="https://github.com/configu/configu/assets/11806915/9cbf458f-5fdc-4135-a2bd-61418d4e05c3">